tests/test_disk_offloader.py

```diff
@@ -81,8 +81,8 @@ def test_async_vec_io(backend):
     except RuntimeError:
         pass
     of.async_writev([x, y])
-    assert x.storage().size() > 0
-    assert y.storage().size() > 0
+    # assert x.storage().size() > 0
```

Comment:
Why?

Reply:
In async_write, we first submit requests then collect completed requests. libaio will have completed requests before collect. so the requests will be collected and the size will be zero.